src/drivers/launch/config/test_cart.yaml

```diff
@@ -0,0 +1,24 @@
+# This file defines physical parameters for the test cart (built for 2017-2018)
+
+# The radius of the wheels
+wheel_radius: 0.13
+
+# The distance between the wheels
+wheelbase_length: 0.66
+
+# The number of ticks for a single rotation of a wheel
+# This will depend on both the number of ticks the encoder reads 
+# and the gear ratio between the encoders and the wheels
+ticks_per_rotation: 3600
+
+# If set to `true` this node assumes that *positive* encoder
+# ticks mean the wheel is moving *forward*. If the encoders are 
+# mirrored, it is likely one of them will need to be reversed
+reverse_left_encoder: false
+reverse_right_encoder: true
+
+# The variance of the left and right encoders.
+# TODO: This could use a more comprehensive comment about what variance is and how to determine it 
+left_encoder_variance: 0.1
```

Comment:
I personally believe that unless we are going past the max read speed, or approaching it, there should not be much variance coming from the sensor itself. If there was any error it would be from rounding and calculations done by the driver and the usb interface, since calculating quadrature should not be tooo bad?

Reply:
the only other case is slipping or mechanical failure, which at that point was supposed to be handled by the EKF anyways.

Reply:
So does this mean the variance on the encoders should be 0? Sorry, bit confused.

Reply:
Basically, I don't think there should should be much variance, unless proven otherwise empirically